lib/tasks/tests-task.ts

```diff
@@ -0,0 +1,27 @@
+import { ITask, ICheckupResult, IProject } from '../../interfaces';
+import Task from '../task';
+import FileSearcher from '../utils/file-searcher';
+
+const SEARCH_PATTERNS = {
+  unit: ['**/tests/unit/**/*.js'],
+  acceptance: ['**/tests/acceptance/**/*.js'],
+  integration: ['**/tests/integration/**/*.js'],
+};
+
+export default class TestsTask extends Task implements ITask {
+  searcher: FileSearcher;
+
+  constructor(project: IProject, result: ICheckupResult) {
+    super(project, result);
+    this.searcher = new FileSearcher(this.project.baseDir, SEARCH_PATTERNS);
+  }
+
+  run(): Promise<ICheckupResult> {
```

Comment:
Can we change this to mirror https://github.com/scalvert/ember-cli-checkup/blob/master/lib/tasks/type-task.ts#L22? This way our tasks will look the same/familiar.